### PR TITLE
Fix glyph.png asset path

### LINKED ISSUE #20894: glyph.png is missing
#### Describe the bug

Active debug mode on prestashop 1.7.7.0-beta2 open console and see error 

We found the directory and the png in prestashop 1.7.6.x  release

But i don't find where the directory **scss** is ignore for the release.

I can  move the file in /admin-dev/themes/new-theme/img/glyph.png

and edit /src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig with the good path ?

**Screenshots**

Capture d’écran de 2020-09-08 14-12-34

#### Additional information

* PrestaShop version: N/A
* PHP version: N/A

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |  1.7.7.x
| Description?  | glyph.png is missing, because scss folder
| Type?         | bug fix 
| Category?     |  BO
| BC breaks?    | no
| Deprecations? |  no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/20894
| How to test?  | Active debug open console

Old issue : https://github.com/PrestaShop/PrestaShop/issues/14757

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20911)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig [hunk 1]
```diff
@@ -43,7 +43,7 @@
     {% endif %}
 
     {% set icon %}
-        <span class="sf-toolbar-label"><img src="{{ asset('themes/new-theme/scss/img/glyph.png') }}" /></span>
+        <span class="sf-toolbar-label"><img src="{{ asset('themes/new-theme/public/glyph.png') }}" /></span>
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig]: The directory `public` is removed each time you compile assets. you must put this file in the `themes/new-theme/img` directory

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Thanks for you pull request, I added a quick comment :)

File: src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig (hunk 1 above)
Comment (PierreRambaud) [reply src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig]: Forget to say, you can remove the file in the scss/img if it's not used in the css :D

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by simobenso — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20911)* status: all files reviewed, 1 unresolved discussion (waiting on @okom3pom)

**Comment by ghost** [comment]
Hi @simobenso 

I'm a beginner I don't understand what to do

**Comment by ngodefroy** [comment]
TEST OK

Capture d’écran 2020-09-18 à 10 40 16

Thank you @okom3pom

**Comment by Progi1984** [comment]
Thanks @okom3pom & @ngodefroy

**Pull request merged**